### DISCUSSION TITLE
Mouse events on geometrical lines created on Mesh

### DISCUSSION THREAD

**Discussion by raksha1991** [pixijs/pixijs#7403] [Q&A]
Hi All,

I am new to Pixijs and WebGL. With the help of one of the community members I could draw millions of lines using Mesh and geometry but now I need to add mouseover and click event for all those lines. I could not find any example for Interactive geometry. Is it possible? I tried to add mousedown event for Mesh and it works fine but I want it for every line created on Mesh.
Working example: https://www.pixiplayground.com/#/edit/wUAri_e9qW6cM9olqeSm7
 
```
mesh.interactive = true;
    mesh.buttonMode = true;
    mesh.hitArea = mesh.getLocalBounds();
    mesh.on("mousedown", function(e) {
        console.log(e);
    });
```

**Comment by eXponenta** [comment]
You should implement intersects manually. You can't use standard methods on non-standard implementation.
Line-mesh is non-standart PIXI object, and we not implemented hit-test for it.
You always can overeide a `containsPoint` by point-to-lint check.

https://en.wikipedia.org/wiki/Distance_from_a_point_to_a_line

But because you have a big amount of lines, you should use a Quad Tree for this, because iterate and calc SQRT of 1 billion lines is super slow operation in naive implemented.
